frappe/public/js/frappe/form/controls/int.js

```diff
@@ -2,21 +2,13 @@ frappe.ui.form.ControlInt = class ControlInt extends frappe.ui.form.ControlData
 	static trigger_change_on_input_event = false;
 	make() {
 		super.make();
-		// $(this.label_area).addClass('pull-right');
-		// $(this.disp_area).addClass('text-right');
 	}
 	make_input() {
-		var me = this;
 		super.make_input();
-		this.$input
-			// .addClass("text-right")
-			.on("focus", function () {
-				setTimeout(function () {
-					if (!document.activeElement) return;
-					document.activeElement.select();
-				}, 100);
-				return false;
-			});
+		this.$input.on("focus", function () {
+			document.activeElement?.select?.();
+			return false;
+		});
```

Comment:
```suggestion
		this.$input.on("focus", () => {
			this.$input?.select?.();
			return false;
		});
```

Reply:
I tried using, this.$input, it doesn't seem to work.

Reply:
To me safe lets use me instead of this.

Reply:
this.$input breaks, so does me.$input. 

me.$input[0] works. I am skipping this change for now. Controls have a long due refactor. :sweat_smile:

Reply:
btw this works fine in Chrome but not in firefox :hankey: